draft-birkholz-scitt-architecture.md

```diff
@@ -162,56 +162,56 @@ Claim:
 
 Issuer:
 
-: creator of Claims submitted to a Transparency Service for Registration. The Issuer may be the owner or author of the Artifact, or a completely independent third party.
+: creator of Claims submitted to a Receipt Service for Registration. The Issuer may be the owner or author of the Artifact, or a completely independent third party.
 
 Envelope:
 
 : the metadata added to the Statement by the Issuer to make it a Claim. It contains the identity of the Issuer and other information to help Verifiers identify the Artifact referred in the Statement. A Claim binds the Envelope to the Statement. In COSE, the Envelope consists of protected headers.
 
-Feed:
+Ledger:
 
-: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Ledger on a Transparency Service contains a sequence of Claims about the same Artifact.
- In COSE, Feed is one header attributes in the protected header of the Envelope.
+: an optional verifiable data structure that stores Claims in a Receipt Service. SCITT supports multiple Ledger formats to accommodate different Receipt Service implementations, such as historical Merkle Trees and sparse Merkle Trees. Not all Receipt Services require a Ledger.
```

Comment:
Making the ledger optional is a significant weakening of the technical guarantee of SCITT, which is principally defined based on the ledger. We think that most of the costs associated with the implementation and operation of SCITT servers are related to the **persistence and distribution/querying of the ledger** rather than its actual computation. For instance, it is possible to implement a minimal append-only ledger using a single hash (like PCRs in TPMs), which technically meets the requirements of append-only and consistent/fork-free. The implementation overhead for such a ledger is only the computation of an extra hash per claim and the persistence of a single hash value in addition to the receipt signing key, while the benefit is a much stronger a clearly defined security goal for SCITT verifiers. I would like to check there is consensus for this change before it is approved.

Reply:
I agree with @ad-l
Do others share Chris' concern?